packages/mongodb-ts-autocomplete/scripts/extract-types.ts

```diff
@@ -0,0 +1,57 @@
+/* eslint-disable no-console */
+import { promises as fs } from 'fs';
+import path from 'path';
+
+function replaceImports(code: string) {
+  // This just makes it possible to work on mql.ts because then the
```

Comment:
Actually my stubbed mql.ts file doesn't even use the bson library at the moment, but this will become relevant again once we have the real mql.ts or otherwise use something that depends on loading in the bson library.